Runtime/Core/AmazonGameLiftPlugin.Core.csproj

```diff
@@ -26,6 +26,9 @@
   </ItemGroup>
   <ItemGroup>
     <!-- See: https://stackoverflow.com/questions/15241889/i-didnt-find-zipfile-class-in-the-system-io-compression-namespace -->
+    <Reference Include="GameLiftServerSDK">
+      <HintPath>..\Plugins\GameLiftServerSDK.dll</HintPath>
+    </Reference>
```

Comment:
Please move after FileSystem reference or before the comment since the comment applies to the FileSystem reference